app/gui/src/project-view/assets/icons.svg

```diff
@@ -1,6 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg">
   <symbol id="3_dot_menu" viewBox="0 0 16 16" width="16" height="16" fill="none">
-    <path fill-rule="evenodd" clip-rule="evenodd" d="M1.33334 3.99999C1.33334 3.63181 1.63182 3.33333 2.00001 3.33333H14C14.3682 3.33333 14.6667 3.63181 14.6667 3.99999C14.6667 4.36818 14.3682 4.66666 14 4.66666H2.00001C1.63182 4.66666 1.33334 4.36818 1.33334 3.99999ZM1.33334 8C1.33334 7.63181 1.63182 7.33333 2.00001 7.33333H14C14.3682 7.33333 14.6667 7.63181 14.6667 8C14.6667 8.36819 14.3682 8.66666 14 8.66666H2.00001C1.63182 8.66666 1.33334 8.36819 1.33334 8ZM1.33334 12C1.33334 11.6318 1.63182 11.3333 2.00001 11.3333H14C14.3682 11.3333 14.6667 11.6318 14.6667 12C14.6667 12.3682 14.3682 12.6667 14 12.6667H2.00001C1.63182 12.6667 1.33334 12.3682 1.33334 12Z" fill="currentColor"></path>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M1.33334 4C1.33334 3.63181 1.63182 3.33334 2.00001 3.33334H14C14.3682 3.33334 14.6667 3.63181 14.6667 4C14.6667 4.36819 14.3682 4.66667 14 4.66667H2.00001C1.63182 4.66667 1.33334 4.36819 1.33334 4ZM1.33334 8C1.33334 7.63181 1.63182 7.33334 2.00001 7.33334H14C14.3682 7.33334 14.6667 7.63181 14.6667 8C14.6667 8.36819 14.3682 8.66667 14 8.66667H2.00001C1.63182 8.66667 1.33334 8.36819 1.33334 8ZM1.33334 12C1.33334 11.6318 1.63182 11.3333 2.00001 11.3333H14C14.3682 11.3333 14.6667 11.6318 14.6667 12C14.6667 12.3682 14.3682 12.6667 14 12.6667H2.00001C1.63182 12.6667 1.33334 12.3682 1.33334 12Z" fill="currentColor"></path>
```

Comment:
I have impression, this icon gets minimal changes on every such PR...

Reply:
Yes I get a lot of minimal changes like this... Can't decide if I care enough to go and look why.